serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -9,22 +9,22 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.2.2-cudnn8-devel-ubuntu22.04
+ARG version=12.4.1-devel-ubuntu22.04
 FROM nvidia/cuda:$version
-ARG cuda_version=cu122
+ARG cuda_version=cu124
 ARG python_version=3.10
-ARG TORCH_VERSION=2.2.1
+ARG TORCH_VERSION=2.2.2
 ARG djl_version=0.28.0~SNAPSHOT
 ARG transformers_version=4.40.0
 ARG accelerate_version=0.29.3
-ARG tensorrtlibs_version=9.3.0.post12.dev1
+ARG tensorrtlibs_version=10.0.1
 ARG trtllm_toolkit_version=nightly
-ARG trtllm_version=v0.9.0
-ARG cuda_python_version=12.2.0
+ARG trtllm_version=v0.10.0_dryrun
+ARG cuda_python_version=12.4
 ARG peft_version=0.10.0
 ARG triton_version=r24.04
 ARG trtllm_toolkit_wheel="https://publish.djl.ai/tensorrt-llm/toolkit/tensorrt_llm_toolkit-${trtllm_toolkit_version}-py3-none-any.whl"
-ARG trtllm_wheel="https://djl-ai.s3.amazonaws.com/publish/tensorrt-llm/${trtllm_version}/tensorrt_llm-0.9.0-cp310-cp310-linux_x86_64.whl"
+ARG trtllm_wheel="https://djl-ai.s3.amazonaws.com/publish/tensorrt-llm/${trtllm_version}/tensorrt_llm-0.10.0-cp310-cp310-linux_x86_64.whl"
 ARG triton_toolkit_wheel="https://publish.djl.ai/tritonserver/${triton_version}/tritontoolkit-24.4-py310-none-any.whl"
 ARG pydantic_version=2.6.1
 ARG ammo_version=0.7.0
```

Comment:
I think Ammo version also changed, need to take a look

Reply:
Also you may want to check the pynvml version

Reply:
The newest pynvml version is still 11.5.0 (and it is the version accepted by TensorRT-LLM)
However, I'm not so sure about the ammo version; I'm not able to find any place in TensorRT-LLM that mentions it

Reply:
You can find more info here: https://github.com/NVIDIA/TensorRT-LLM/blob/main/examples/quantization/README.md#preparation

They seemed to change the name to

```
pip install --no-cache-dir --extra-index-url https://pypi.nvidia.com nvidia-modelopt==0.9.3
```

nvidia-modelopt => ammo

Reply:
Nice catch, fixed